river/cluster/denstream.py

```diff
@@ -105,8 +109,8 @@ class DenStream(base.Clusterer):
     ... ]
 
     >>> denstream = cluster.DenStream(decaying_factor = 0.01,
-    ...                               beta = 1.01,
```

Comment:
Formatting is a bit weird here. We usually don't use spaces around operators

Reply:
@MaxHalford Sorry I definitely did not notice this upon fixing. Will be modified later!

Reply:
Resolved!